src/modules/user/model/index.js

```diff
@@ -40,7 +40,14 @@ const registrationSchema = {
   password: ['required', 'string', 'confirmed', 'password_validations'],
 };
 
+const loginSchema = {
+  email: ['required', 'string', 'email'],
+  password: ['required', 'string', 'confirmed', 'password_validations'],
```

Comment:
For password at login we don't need to validate confirmed, or the other password validations rules since they will be checked on registration and reset-password

Reply:
fixed